roadmap.html

```diff
@@ -36,9 +36,13 @@ <h3><a href="https://github.com/neovim/neovim/milestone/19">0.5</a></h3>
   <li>Extended marks (text properties, decorations, virtual text)</li>
```

Comment:
i don't think 0.5 has Lua remote plugin host, so that should be moved to 0.6 also?